system/settings/settings.xml

```diff
@@ -2192,12 +2192,6 @@
           <control type="spinner" format="string" />
         </setting>
         <setting id="videoscreen.limitedrange" type="boolean" label="36042" help="36359">
-          <requirement>
-            <or>
-              <condition>HAS_GL</condition>
-              <condition>HAS_DX</condition>
-            </or>
-          </requirement>
```

Comment:
Some settings of AE can serve as an example or this one: https://github.com/FernetMenta/kodi-agile/blob/master/system/settings/settings.xml#L200